ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -343,46 +343,44 @@ then
 		if [ -f /etc/extras/px4io-v2.bin ]
 		then
 			set IO_FILE /etc/extras/px4io-v2.bin
```

Comment:
@LorenzMeier  - pleas have a look at this. section. The assumption is that if the "/etc/extras/px4io-v2.bin" does not exist the PX4IO is not started. This is in contrast to using the "/etc/extras/px4io-v1.bin" that might not have existed and failing on each ${IO_FILE} access.